starwars/src/components/Characters.js

```diff
@@ -0,0 +1,47 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import styled from "styled-components";
+import CharacterCards from "./CharacterCards";
+
+
+const StyledCard = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;    
+    margin: 1%;
+   
+`;
+
+export default function Characters() {
+    const [characters, setCharacters] = useState([]);
```

Comment:
Great use of state here. Initializing state as an array and setting it to an array of objects is the best way to do this. Very good variable names. Makes a lot of sense here.